estuary/utils/general.py

```diff
@@ -366,3 +352,32 @@ def format_story_results(results, requested_item):
             'requested_node_index': requested_node_index
         }
     }
+
+
+def organize_labels(requested_node_label, results, reverse=False):
+    """
+    Replace Neo4j labels with appropriate labels of the story flow.
+
+    :param string requested_node_label: label of the node requested by the user
+    :param list results: nodes in a story/path
+    :kwarg bool reverse: determines if the results are in reverse order of the story flow
+    :return: results with story/path labels
+    :rtype: list
+    """
+    # Avoid circular imports
+    from estuary.models import story_flow_list
+
+    if results:
```

Comment:
Cosmetic.  This could become

```python
if not results:
    return results

idx = ...
```

... which would allow you to de-indent the entire function.